.github/workflows/mops-publish.yml

```diff
@@ -1,8 +1,9 @@
 name: publish on mops
 
 on:
-  release:
-    types: [released]
+  push:
+    tags:
+      - moc-*
```

Comment:
If I understand correctly, this would attempt to publish a Mops package each time we merge a PR (including this one). Is that the intent? 

I'm doing something similar with the `prettier-plugin-motoko` binaries, where instead of using tags it simply looks for a new version in the `package.json` file. We could do something like that for publishing to Mops, but it would probably require some additional setup.

Reply:
It looks like the `release` job only runs when creating a new tag: [`if: startsWith(github.ref, 'refs/tags/')`](https://github.com/dfinity/motoko/blob/3f3af7390302a955b01b0537e0a0f6941f153b72/.github/workflows/release.yml#L72)

My understanding (from skimming through the file) is that some of the other jobs run for both new tags and the main branch, whereas the full release workflow only runs when creating a tag. 

> We could do something like that here, but it would probably require some additional setup.

This refers to automatically generating the tags whenever we push a new version in the `mops.toml` file, rather than manually creating them for each release. Probably makes sense to do this in a separate PR (if at all).

Reply:
Sorry, I have close to no understanding of the yml files. I just though the idea was to only publish to mops when we manually create and push new tag of the form `moc-*`. But it looks like I don't full understand what is going on.

Reply:
FTR, we never made proper GH release of base in the past. Instead, when we release Motoko, we tag the head of motoko base branch `next-moc` and then merge next-moc into master.
See here https://github.com/dfinity/motoko/blob/master/Building.md#making-releases (towards the end of the section).

Reply:
Just to double-check, we manually push the tags in `motoko-base` for each compiler version (as mentioned in the `Building.md` instructions), right? If so, the PR should work as intended in its current state.

I'll also push a change so that we can manually trigger the CI workflow in case we run into any issues for the next release.